src/sonic-bgpcfgd/tests/data/voq_chassis/instance.conf/param_all.json

```diff
@@ -0,0 +1,24 @@
+{
+    "CONFIG_DB__DEVICE_METADATA": {
+        "localhost": {
+            "sub_role": "BackEnd"
```

Comment:
In "voq" based systems, all asics (except fabric asics) are considered "FrontEnd" and unless we run BGP in fabric asics, we would not need a "BackEnd" sub-role in voq systems.